libexec/scoop-info.ps1

```diff
@@ -76,66 +74,63 @@ if ($manifest.license) {
     } else {
         $license = "$($manifest.license) (https://spdx.org/licenses/$($manifest.license).html)"
     }
-    Write-Output "License: $license"
+    $item.License = $license
 }
 
 # Manifest file
-Write-Output "Manifest:`n  $manifest_file"
+$item.Manifest = $manifest_file
 
-if($status.installed) {
+if ($status.installed) {
     # Show installed versions
-    Write-Output "Installed:"
-    $versions = Get-InstalledVersion -AppName $app -Global:$global
-    $versions | ForEach-Object {
-        $dir = versiondir $app $_ $global
-        if($global) { $dir += " *global*" }
-        Write-Output "  $dir"
+    $installed_output = ""
+    Get-InstalledVersion -AppName $app -Global:$global | ForEach-Object {
+        $installed_output += "`n$(versiondir $app $_ $global)"
     }
-} else {
-    Write-Output "Installed: No"
+    $item.Installed = $installed_output
 }
 
 $binaries = @(arch_specific 'bin' $manifest $install.architecture)
-if($binaries) {
-    $binary_output = "Binaries:`n "
+if ($binaries) {
+    $binary_output = ""
     $binaries | ForEach-Object {
-        if($_ -is [System.Array]) {
-            $binary_output += " $($_[1]).exe"
+        if ($_ -is [System.Array]) {
+            $binary_output += "`n$($_[1]).exe"
         } else {
-            $binary_output += " $_"
+            $binary_output += "`n$_"
         }
     }
-    Write-Output $binary_output
+    $item.Binaries = $binary_output
 }
 $env_set = (arch_specific 'env_set' $manifest $install.architecture)
 $env_add_path = (arch_specific 'env_add_path' $manifest $install.architecture)
-if($env_set -or $env_add_path) {
-    if($status.installed) {
-        Write-Output "Environment:"
-    } else {
-        Write-Output "Environment: (simulated)"
-    }
-}
-if($env_set) {
+if ($env_set) {
+    $env_vars = ""
     $env_set | Get-Member -member noteproperty | ForEach-Object {
         $value = env $_.name $global
-        if(!$value) {
+        if (!$value) {
             $value = format $env_set.$($_.name) @{ "dir" = $dir }
         }
-        Write-Output "  $($_.name)=$value"
+        $env_vars += "`n$($_.name) = $value"
     }
+    $item.'Environment Variables' = $env_vars
 }
-if($env_add_path) {
+if ($env_add_path) {
+    $env_path = ""
     $env_add_path | Where-Object { $_ } | ForEach-Object {
-        if($_ -eq '.') {
-            Write-Output "  PATH=%PATH%;$dir"
+        if ($_ -eq '.') {
+            $env_path += "`n$dir"
         } else {
-            Write-Output "  PATH=%PATH%;$dir\$_"
+            $env_path += "`n$dir\$_"
         }
     }
+    $item.'Path Additions' = $env_path
```

Comment:
```suggestion
    $item.'Path Appended' = $env_path
```

Reply:
Append means adding at the end, which is not true. Scoop actually "prepends" the PATH variable. But in the end, it becomes somewhere in between because the machine wide PATH takes a priority. So, it is ambiguous and hence "Addition".

Reply:
So why not 'Path Added'? Path Addition sounds strange.

Reply:
The property name should be a noun, right? "Added" is verb, "Addition" is noun.

Reply:
😂 Path added means 'Path to be added', it's noun.